src/Compilers/Core/Portable/Compilation/GeneratedKind.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis
+{
+    public enum GeneratedKind
+    {
+        /// <summary>
+        /// Whether the <see cref="SyntaxTree"/> is automatically generated is unknown.
```

Comment:
```suggestion
        /// It is unknown if the <see cref="SyntaxTree"/> is automatically generated.
```

The "Whether X is Y is Z" phrasing really confused me.